packages/manager/apps/pci-kubernetes/src/components/radio-tile/RadioTile.component.tsx

```diff
@@ -49,7 +49,7 @@ const RadioTile = ({
         <label
           id={labelId}
           className={cn(
-            'flex flex-col  h-full w-full group text-primary-600 border-2 border-primary-100 rounded-md',
+            'flex flex-col  h-full w-full group  border-2 border-primary-100 rounded-md',
```

Comment:
You can remove unnecessary double spaces 
```suggestion
            'flex flex-col h-full w-full group border-2 border-primary-100 rounded-md',
```